lib/FlashMem/FlashMem.h

```diff
@@ -72,6 +72,12 @@ namespace Flash
     /** Maximum length for saved Credentials. */
     static const uint8_t CREDENTIALS_MAX_LENGTH = NVM_PASSWORD_ADDRESS + NVM_PASSWORD_MAX_LENGTH;
 
+    /** Address of saved Groups in EEPROM. */
+    static const uint8_t NVM_GROUPS_ADDRESS = CREDENTIALS_MAX_LENGTH;
+
+    /** Length of saved Groups in EEPROM. */
+    static const uint8_t NVM_INTEGER_LENGTH = 1;
```

Comment:
Strange name ... stay consistent all the time! Expected would be `NVM_GROUPS_LENGTH`, isn't it?